tip-xxxx/tiip-did-jp.md

```diff
@@ -0,0 +1,368 @@
+```
```

Comment:
After all, I thought it would be better without japanese for the following reasons.

* If the content differs between English and Japanese, there is a risk that specifications will not be accurately communicated.
* Updating the TIP requires updating multiple language documents.
* Reviewing both English and Japanese is a burden on the reviewer.